kernel/src/actions/mod.rs

```diff
@@ -28,6 +28,8 @@ pub(crate) const METADATA_NAME: &str = "metaData";
 pub(crate) const PROTOCOL_NAME: &str = "protocol";
 pub(crate) const SET_TRANSACTION_NAME: &str = "txn";
 pub(crate) const COMMIT_INFO_NAME: &str = "commitInfo";
+#[allow(unused)]
```

Comment:
I assume when we're ready to go here this allow won't be needed?

Reply:
Yep, I'll go ahead and add a comment